teleprox/client.py

```diff
@@ -159,14 +188,16 @@ def __init__(self, address, reentrant=True, start_local_server=False, serializer
 
             # For unserializing results returned from servers. This cannot be
             # used to send proxies of local objects unless there is also a server
```

Comment:
The server has its own serializer, though, so can we just say this cannot be used yo send proxies of local objects?